README.md

```diff
@@ -77,9 +83,12 @@ and emit a new state indicating that these records are in flight.
 ```javascript
 state.length //=> 10
 let record = state.get(7)
+record.isRequested //=> true
 record.isPending //=> true
 record.isResolved //=> false
+record.index //=> 2
 record.content //=> null
+record.page //=> { offset: 1, size: 5, data: [null*5] }
 ```
```

Comment:
Changed a Pending Page `page.data[0]; //=> null` instead of `page.data[0]; //=> {}` because `record.content; //=> null`. Therefore it maps directly.